### PR TITLE
Math magicians components :- module 3

### PR DESCRIPTION
**Pull request by yusufyusie**
### Project requirements
In the previous project, you worked on the setup of a React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html), now you should:

- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
- Implement the design only (JSX and CSS)

- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use functional components
- Create at least 1 child component
- Once the Calculator is ready, import it and use it in your top-level component: App.js
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    );
+  });
+
+  return (
+    <div className="wrapper">
+      {' '}
+      <div className="show-input">{input}</div>
+      <div className="digits flex">{calcBtns}</div>
+      <div className="modifiers subgrid">
+
```
Comment (Owusu-Desmond) [inline src/components/Calculator.js]: - [ ] Please ensure that the calculator component has at least one child component. I recommend creating a `<Button/>` child component to meet the project requirements. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @yusufyusie , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      {/* "=" btn */}
+      <button
+        type="button"
+      >
+        =
+      </button>
+    </div>
+  );
+}
+export default Button;
```
Comment (Strangeal) [inline src/components/Button.js]: - [ ] Great job creating a `child component` as suggested by the previous reviewer. However, I can see you're not making use of this child component anywhere in your project which is more like you not creating it at all. Kindly make use of the `child component` you have created.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @yusufyusie, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @yusufyusie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**